tox.ini

```diff
@@ -1,12 +1,16 @@
 [tox]
-envlist = py{27,35,36}
-skip_missing_interpreters = true
+envlist = py27, py35, py36
 
 [testenv]
+usedevelop = True
 deps =
-  -rrequirements.txt
-  -rdev-requirements.txt
-usedevelop = true
-commands =
-  pip freeze
-  pytest -vv test {posargs}
+    -rrequirements.txt
+    -rdev-requirements.txt
+commands = pytest -vs --cov gdcdatamodel --cov-report xml {posargs}
+
+[testenv:coverage]
```

Comment:
don't you need to have this in your `envlist`?

Reply:
I could target it directly without specifying it. And for some reason the envlist doesn't work as expected when I do `tox`